scripts/stop_sf_executions.py

```diff
@@ -0,0 +1,23 @@
+import boto3
+
+
+if __name__ == "__main__":
+    boto3.setup_default_session(profile_name="KlayersProdP38")
+
+    client = boto3.client('stepfunctions')
+    response = client.list_executions(
+        stateMachineArn='arn:aws:states:us-east-2:770693421928:stateMachine:kl-Klayers-prodp38-pipeline',
+        statusFilter='RUNNING',
+    )
+
+    if len(response['executions']) > 0:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0`

[Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)